tests/Duckling/Time/Tests.hs

```diff
@@ -82,8 +82,8 @@ timeFormatTest :: TestTree
 timeFormatTest = testCase "Format Test" $
   mapM_ (analyzedFirstTest testContext testOptions . withTargets [Seal Time]) xs
   where
-    xs = examplesCustom (parserCheck expected parseValue) ["now"]
-    expected = "2013-02-12T04:30:00.000-02:00"
```

Comment:
So I think we on purposely do not want to handle "now" using duckling right?

Reply:
Yeah I made "now" latent with a previous release. The problem is that it is used often on updates from the user outside of the context of a time related to the query: 

1. "show my transactions from Walmart last month"
2. "now show my transactions from Kroger"

We'd want to maintain "last month" in this case. "now" also tends to be a default for actions (e.g. transfers) so it's safer to not extract than to extract erroneously